opentelemetry/proto/collector/logs/v1/logs_service.proto

```diff
@@ -43,4 +43,26 @@ message ExportLogsServiceRequest {
 }
 
 message ExportLogsServiceResponse {
+  // The details of a partially successful export request.
+  // If the processing of the request is partially successful
+  // (i.e. when the server accepts only parts of the data and rejects the rest)
+  // and the number of accepted log records is known, the server MUST initialize
+  // the partial_success field and MUST set the accepted_log_records.
+
+  // When the export request is processed successfully, the server MUST leave
+  // the partial_success field unset.
+  ExportLogsPartialSuccess partial_success = 1;
+}
+
+message ExportLogsPartialSuccess {
```

Comment:
Are we attempting to report partial failure so... we can retry problematic logs?

Or is this more about NOT retrying e.g. a "log of death"?

Specifically, should this include some notion of WHICH logs were rejected?

Reply:
> Or is this more about NOT retrying e.g. a "log of death"?

More or less on this I'd say. 

It's about informing that a server accepted only parts of the data. The why it happened can be due to N reasons and depends highly on the server, so no attempt of retry is defined for now. The current state of this PR can also allow senders to "selfmonitor" and report the number of accepted things as a metric for example.

Also no attempt to indicate which logs/datapoints/spans the server rejected, because that is still not clear how it will be modeled and also could be a burden on receivers under pressure for ex. There was a discussion around this here https://github.com/open-telemetry/opentelemetry-proto/pull/390#discussion_r871989856

> Specifically, should this include some notion of WHICH logs were rejected?

This is also related to the discussions I linked above. We agreed that including more info to determine which logs/spans/data points were rejected would make sense but we need to discuss it better since it comes with its own set of challenges. I suggested creating separated issues to discuss it per sinal to not block this. There's one for metrics here https://github.com/open-telemetry/opentelemetry-proto/issues/404